resources/views/components/input/input.blade.php

```diff
@@ -14,4 +14,4 @@
 </label>
 ```
 --}}
-<input {{ $attributes->twMerge('w-full py-3 px-5 border border-default rounded-md outline-0 ring-0 text-sm transition-colors focus:ring-transparent focus:border-emphasis disabled:cursor-not-allowed disabled:bg-muted disabled:border-muted placeholder:text-inactive [&::-webkit-inner-spin-button]:appearance-none [&:user-invalid:not(:placeholder-shown)]:border-red-500') }} />
+<input {{ $attributes->twMerge('w-full py-3 px-5 border border-default rounded-md outline-0 ring-0 text-sm transition-colors focus:ring-transparent focus:border-emphasis disabled:cursor-not-allowed disabled:bg-emphasis disabled:border-emphasis placeholder:text-inactive [&::-webkit-inner-spin-button]:appearance-none [&:user-invalid:not(:placeholder-shown)]:border-red-500') }} />
```

Comment:
Why do you use `emphasis` and not `muted`?

Reply:
Because the default bg is already a very light color, when using muted of that bg variant you use an even lighter color of an already light color. That in combination with a white <body background makes the contrast nearly invisible. The emphasis gives it a more disabled look / not allowing to be interactive.

Reply:
Maybe disabled should be it's own color. What do you think?

Reply:
> Maybe disabled should be it's own color. What do you think? disabled:bg-disabled?

That could be an option. 

Can you also check the placeholder color? I see an variable `text-inactive`.